bolt-design-system/bolt
# Fix/horizontal scroll from character

## Jira

https://app.asana.com/0/1126340469288208/1137915602201453

## Summary

Fix content overflow; responsiveness for bolt-connection and bolt-character

## How to test

Check out micro-journey in band. Check different widths. Make sure to refresh after scaling viewport upward.
